java-client/src/main/java/com/cumulocity/sdk/client/event/EventFilter.java

```diff
@@ -39,6 +40,10 @@ public class EventFilter extends Filter {
     @ParamSource
     private String fragmentType;
 
+    @ParamSource
+    @Getter
+    private String fragmentValue;
```

Comment:
I would stick to the convention of directly specifying getters in this file. 
There may be something regarding generating javadocs from lombok in this project.